### ISSUE TITLE
Allow to specify keyword arguments for HTTPClient when creating instance of ESConnection

### ISSUE THREAD

**Issue by pawelmhm** [globocom/tornado-es#39]
Current API does not allow to specify parameters for Tornado.AsyncHTTPClient used under the hood by Tornado-es when connecting to ES. 

I stumbled on this because I'm debugging Tornado app and noticed warnings in logs

```
max_clients limit reached, request queued. 10 active, 11 queued requests.
```

these warnings are emitted by AsyncHTTPClient. I would like to adjust number of concurrent connections. Base Tornado client has variety of parameters that can be passed see [here](https://github.com/tornadoweb/tornado/blob/master/tornado/simple_httpclient.py#L59). If you look at [usage of client in tornado-es](https://github.com/globocom/tornado-es/blob/master/tornadoes/__init__.py#L18) you can see that you can't pass anything to http client.

This would be just matter of adding some extra arguments to `__init__` and passing them to client. Is it ok if I create PR with that?

cc @luizgpsantos

**Comment by pawelmhm** [comment]
fixed via: #38

**Comment by pawelmhm** [comment]
One thing to keep in mind is that tornado.AsyncClient does not reuse TCP connections. tornado.CurlAsyncHTTPClient reuses TCP connections. CurlClient might be better default option, I noticed some performance improvement when using CurlAsyncHTTPClient.
